editor/block-mover/test/index.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import { BlockMover } from '../';
+
+describe( 'BlockMover', () => {
+	describe( 'basic rendering', () => {
+		it( 'should render two IconButton components with the following props', () => {
+			const blockMover = shallow( <BlockMover /> );
+			expect( blockMover.hasClass( 'editor-block-mover' ) );
+
+			const moveUp = blockMover.childAt( 0 );
+			const moveDown = blockMover.childAt( 1 );
+			expect( moveUp.type().name ).to.equal( 'IconButton' );
+			expect( moveDown.type().name ).to.equal( 'IconButton' );
+			expect( moveUp.props() ).to.include( {
+				className: 'editor-block-mover__control',
+				onClick: undefined,
```

Comment:
Okay, looking at this again, I think I see what's happening. I assumed the test case was asserting on the `null` value passed to `onClick`, and that `includes` assertion wasn't smart enough to differentiate between a value being explicitly undefined, vs. not making an assertion on that value.

Instead, `includes` _will_ make this assert that the value is explicitly `undefined`, and it's not the `null` value we're expecting here, it's `onMoveUp`, which in this case is `undefined` because we're not passing this prop to the component in the test.

Which is all to say... this is fine 😄